app/src/components/common/text/subsection_title_wrapper/index.tsx

```diff
@@ -6,7 +6,7 @@ const Wrapper = styled.div`
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
-  margin-bottom: 20px;
+  margin-bottom: 24px;
```

Comment:
There are quite a few places in which this change is represented. @pimato is this intended behaviour?